CppCoreGuidelines.md

```diff
@@ -20956,7 +20956,7 @@ Most of the concepts below are defined in [the Ranges TS](http://www.open-std.or
 * `String`   // ???
 * `Number`   // ???
 * `Sortable`
-* `EqualityComparable`   // ???Must we suffer CaMelcAse???
+* `EqualityComparable`
```

Comment:
technically we no longer "must suffer" it, since C++20 concepts are in snake_case, but that is going to be a bigger update, #1547